### DISCUSSION TITLE
problem grabbing epg

### DISCUSSION THREAD

**Discussion by kaball76** [iptv-org/community#1007] [Q&A]
hi i am trying to grab the epg from directv. i am using a vpn connected to USA but i get  error (see image)
![Screenshot 2023-12-30 185244](https://github.com/iptv-org/iptv-org.github.io/assets/155222778/3e959f38-c86d-48c5-8e9f-5142f2bd7266)
and when i try tvtv.us i get the below error
![Screenshot 2023-12-30 185506](https://github.com/iptv-org/iptv-org.github.io/assets/155222778/33b11e91-41aa-4ab9-afe8-c938cea24c8d)
can someone please help me

**Comment by BellezaEmporium** [comment]
As for tvtv.us, it has gone onto a recent change, so it should have been fixed. I do not know what's the issue for directv.com. Maybe an API change.

**Comment by PopeyeTheSai10r** [comment]
DirecTV started blocking a bunch of known VPN IP Address see if you can navigate to https://www.directv.com/guide/channel-guide from a browser while connected to the VPN.

**Reply by BellezaEmporium** [reply]
![image](https://github.com/iptv-org/iptv-org.github.io/assets/30985701/e91e9c96-4941-4c95-bc9d-06dcce9dfe4d)
with US VPN (New York)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/epg/issues/2284
